docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1037,7 +1037,7 @@ How to **investigate**:
   1. The alert fired because of a bug in Mimir: fix it.
   1. The alert fired because of a bug or edge case in the continuous test tool, causing a false positive: fix it.
 
-## Codified errors
+## Errors catalog
```

Comment:
@zenador This change belongs to separate PR.

Reply:
Moved to https://github.com/grafana/mimir/pull/2256